PactNet/Mocks/MockHttpService/AdminHttpClient.cs

```diff
@@ -82,22 +80,12 @@ public async Task<string> SendAdminHttpRequest(HttpVerb method, string path, IDi
                 throw new PactFailureException(responseContent);
             }
 
-            if (!string.IsNullOrEmpty(responseContent))
-            {
-                return responseContent;
-            }
-            else
-            {
-                return string.Empty;
-            }
+            return !string.IsNullOrEmpty(responseContent) ? responseContent : Empty;

```

Comment:
Since you have used `using static System.String;`, you may also consider using it in terms of `IsNullOrEmpty`